### DISCUSSION TITLE
Creator Coin sale and transfer notifications

### DISCUSSION THREAD

**Discussion by oO** [deso-protocol/dips#193] [Medium CIPs]
I find it very strange that we have notifications when someone buys your coin, but not when coins are sold or transferred. Do we know if this is an omission, or if there is a specific reason why Someone buying your coin generates a notification, but someone selling your coin doesn't?

**Comment by darian-parrish** [comment]
I don't know of any specific reason why not other than it could be called a "negative motivation" notification whereas other notifications are supposed to boost morale.

This is more of an issue for the app layer though rather than the core code or protocol since these transaction types do exist.

**Reply by oO** [reply]
IMHO the backend API should generate all the notifications, it's up to the front end implementations to decide if they want to implement them or not.

**Reply by darian-parrish** [reply]
I mean, it would definitely be nice if it were returned by the get notifications endpoint but the question becomes is it something most apps will use. Agree that not all need to use it but there normally has to be some sort of demand threshold whether that is 50% or more I don't know how the core team is thinking about it.
